dango/vesting/src/state.rs

```diff
@@ -0,0 +1,16 @@
+use {
+    dango_types::vesting::Position,
+    grug::{Addr, Counter, IndexedMap, MultiIndex},
+};
+
+pub const POSITION_INDEX: Counter<u64> = Counter::new("position_index", 0, 1);
```

Comment:
As a convention, we always name indexes `NEXT_` something, e.g.

- `NEXT_ACCOUNT_INDEX` in dango-account-factory
- `NEXT_PROPOSAL_ID` in dango-account-safe
- `NEXT_SEQUENCE` in dango-auth
- `NEXT_POOL_ID` in dango-amm

Reply:
Also is it necessary to use `u64`?

`u32` should be sufficient.